genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/JobExecutionStateMachine.java

```diff
@@ -38,4 +38,14 @@
      * @throws InterruptedException if the waiting thread is interrupted
      */
     States waitForStop() throws InterruptedException;
+
+    /**
+     * Force an early termination of the state machine, in response to user submitting a kill via API or ctrl-c.
+     * Notice:
+     *  - State actions are not interrupted. Only after the one currently executing is completed the stop
+     *    signal is processed (TODO: make long-running actions, such as setup, interruptible).
+     *  - Some actions are still performed before the program exits. For example publish the updated job status
+     *    server-side. Refer to the state machine configuration for details.
```

Comment:
Is there a `@see` or `@link` on this instead of just saying "refer" so we quickly know where to go?

Reply:
👍